### PR TITLE
Introduce BINDIR, LIBDIR, INCLUDEDIR and DESTDIR

### PR DESCRIPTION
**Pull request by jforissier**
Signed-off-by: Jerome Forissier <jerome.forissier@linaro.org>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -122,11 +125,11 @@ checkpatch-all-files: checkpatch-pre-req
 distclean: clean
 
 copy_export: build
-	mkdir -p ${EXPORT_DIR}/lib ${EXPORT_DIR}/include ${EXPORT_DIR}/bin
-	cp ${O}/libteec/libteec.so* ${EXPORT_DIR}/lib
-	if [ "$(BUILD-LIBSQLFS)" ]; then cp ${O}/libsqlfs/libsqlfs.so* ${EXPORT_DIR}/lib; fi
-	cp ${O}/tee-supplicant/tee-supplicant ${EXPORT_DIR}/bin
-	cp public/*.h ${EXPORT_DIR}/include
+	mkdir -p $(DESTDIR)$(BINDIR) $(DESTDIR)$(LIBDIR) $(DESTIDIR)$(INCLUDEDIR)
```
Comment (jbech-linaro) [inline Makefile]: Where is DESTDIR defined?

Comment (jbech-linaro) [inline Makefile]: Exactly, that was what I was looking for. Having that said, I've tested it too:
`Reviewed-by: Joakim Bech <joakim.bech@linaro.org>`
`Tested-by: Joakim Bech <joakim.bech@linaro.org> (RPi3)`

Comment (jforissier) [inline Makefile]: @jbech-linaro I have updated the commit to define `DESTDIR` as above, would you mind checking again?

**Comment by jbech-linaro** [comment]
Everything is fine with last changes
`Reviewed-by: Joakim Bech <joakim.bech@linaro.org>`
`Tested-by: Joakim Bech <joakim.bech@linaro.org> (RPi3)`

**Comment by jforissier** [comment]
Thanks!

**Pull request closed without merging**